routers/rooms.py

```diff
@@ -20,11 +20,11 @@ async def create_rooms_type(roomtype: rooms_schema.RoomTypeCreate):
         Returns:
                 Dict with result
     """
-    return await rooms_utils.create_room_type(roomtype=roomtype)
+    return await rooms_crud.create_room_types(roomtype=roomtype)
```

Comment:
if you create a single entity use singular nouns in function names

Reply:
ok